src/algo/camera-age/l500/camera-age.h

```diff
@@ -4,33 +4,36 @@
 #pragma once
 
 #include <string>
+#include <ctime>
 
 
 namespace librealsense {
 namespace algo {
 namespace camera_age {
 namespace l500 {
 
-class manufacture_time
+class work_week
 {
     unsigned man_year;
```

Comment:
Not sure what `man` stands for. But let's follow the convention in LRS that members start with `_`